configGenerator.js

```diff
@@ -51,12 +51,14 @@ try {
   // Write the substituted data to config.json
   try {
     fs.writeFileSync(currentConfigFilePath, substitutedData, 'utf-8');
-    console.log('Configuration file written in current directory.');
+    // eslint-disable-next-line no-console
+    console.log('Configuration file written in current directory.'); // NOSONAR
```

Comment:
same is the necessary

Reply:
Same for this this is only during development.